core/src/test/scala/unit/kafka/server/ApiVersionManagerTest.scala

```diff
@@ -40,10 +40,15 @@ class ApiVersionManagerTest {
       listenerType = apiScope,
       forwardingManager = None,
       features = brokerFeatures,
-      metadataCache = metadataCache
+      metadataCache = metadataCache,
+      enableUnstableLastVersion = true
```

Comment:
Does it make sense to add a test case when this flag is false. We could iterate the api keys and look for the presence of the `latestVersionUnstable` flag and then assert that the api is not enabled.

Reply:
That makes sense.

Reply:
+1